specification/security/resource-manager/readme.md

```diff
@@ -414,15 +427,6 @@ These settings apply only when `--tag=package-composite-v3` is specified on the
 
 ``` yaml $(tag) == 'package-composite-v3'
 input-file:
-- Microsoft.Security/preview/2021-10-01-preview/mdeOnboardings.json
```

Comment:
why you remove these resource types?

Reply:
These were all duplicates - you can see that they are still available under the `package-composite-v3` tag.